### DISCUSSION TITLE
All settings of disable ads

### DISCUSSION THREAD

**Discussion by p1r473** [SeaHOH/ban-peers#3] [Show and tell / 展示]
Hello
Here are all of the settings I have found to disable ads
Some of them are missing from the tool

```
bt.enable_pulse false
distributed_share.enable false
gui.show_notorrents_node false
offers.left_rail_offer_enabled false
gui.show_plus_upsell false
offers.content_offer_autoexec false
offers.sponsored_torrent_offer_enabled false
offers.featured_content.badge.enabled false
offers.featured_content_notifications_enabled false
offers.featured_content_rss_enabled false

Hidden menu = Options > Hold F2 and Shift while clicking Peferences
gui.show_gate_notify false
gui.show_plus_av_upsell false
gui.show_plus_conv_upsell false
gui.show_plus_upsell_nodes false
gui.show_gate_explaination false
offers.backup_left_rail_offer_enabled false
offers.backup_sponsored_torrent_offer_enabled false
offers.upgrade_panel false
offers.upgrade_toolbar false

Hiding bundles
Right click->hide featured content
```

**Comment by SeaHOH** [comment]
Thanks so much.
Do you means `offers.upgrade_panel` is missing? I didn't found it in newest build, which build is it still in?

>     Hiding bundles
>     Right click->hide featured content

I think this setting have to be set manually, right?

And, are you interested in [this guide](https://github.com/SeaHOH/ban-peers/issues/1) translated from Chinese into English, it guide how to modify the Windows uTorrent.exe file. Maybe we can put it in Wiki page?

**Comment by p1r473** [comment]
> Do you means `offers.upgrade_panel` is missing? I didn't found it in newest build, which build is it still in?
> 
> > ```
> > Hiding bundles
> > Right click->hide featured content
> > ```

I am not sure which features you are missing- I do not know all of the features you change
I have included all the settings I know to change
If any are missing from the list, please include it

**Comment by p1r473** [comment]
> > > ```
> > > Hiding bundles
> > > Right click->hide featured content
> > > ```
> > 
> > 
> > I think this setting have to be set manually, right?

Yes this one is manual

**Comment by SeaHOH** [comment]
Those are using settings right now.
https://github.com/SeaHOH/ban-peers/blob/bcc7fafb9e9c0fe6d473309a4813e92d8b1ad24a/ban_peers.py#L176-L235

**Comment by SeaHOH** [comment]
I found uTorrent 3.5.5 build 44876, the last build of `offers.upgrade_panel`, released on 2018-12-03. That's almost two years now, I think we don't need it.

**Comment by p1r473** [comment]
Sorry, I am unsure which build.

**Comment by SeaHOH** [comment]
I would be working on new builds (restrict to anti-ads, it's mutable), because I don't want to test a large number of old builds.

**Comment by loading1688** [comment]
博主你好！能不能做个成品！这样太费劲！那个去广告研究2天也没安装上 还是算了吧！谢谢

**Comment by SeaHOH** [comment]
你指的是修改程序本体那个指南？我当然不会提供成品，不然岂不是自相矛盾嘛。
哪里不明白可以在指南下留言，我会尽量讲解清楚。

不过如你确实需要现成的， www.423down.com 提供的修改版我对比过本体，至少现在应该是没有问题的。
其中 settings.dat 文件没有仔细看过，不一定要用它，可以换成空白文件，然后在断网情况下使用 `ban_peers -A`。
